### PR TITLE
Add more regions to registry S3 storage driver

### PR DESCRIPTION
**Pull request by mfojtik**
Signed-off-by: Michal Fojtik <mfojtik@redhat.com>

### CODE REVIEW

**Comment by dmcgowan** [comment]
LGTM

Looks like there are at least 2 others not in the list, `ap-south-1` and `eu-west-2`

**Comment by mfojtik** [comment]
@dmcgowan I can add them as well here, thx for noticing.

**Comment by pweil-** [comment]
LGTM, thanks @mfojtik

**Comment by dmcgowan** [comment]
LGTM

**Comment by mfojtik** [comment]
(note that I haven' tested this yet and I'm worried that I can't grep these endpoint names in aws-sdk-go)

**Comment by dmcgowan** [comment]
Might need to upgrade the dependency to 1.6.3 then https://github.com/aws/aws-sdk-go/commit/ad048b5f9e1187bee316f6919971c0e122d701e4

Looks like ap-south-1 and ca-central-1 were added in 1.6.2, eu-west-2 in 1.6.3  https://github.com/aws/aws-sdk-go/commit/1355b456f6ba4a8453249e0dc7743f16ccac362d

**Comment by dmcgowan** [comment]
Looks like upgrading the dependency will pull in a lot more changes, I am going to move this back to 2.7 release. Maybe could consider for 2.6.1 if it is important for you guys and you have time to test it.

**Comment by mfojtik** [comment]
@dmcgowan i bumped aws dep to 1.6.3 and we will do some testing in OpenShift to make sure it works well. The distribution tests seems to pass.

@miminar @legionus PTAL

File: vendor.conf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 github.com/Azure/azure-sdk-for-go c6f0533defaaaa26ea4dff3c9774e36033088112
 github.com/Sirupsen/logrus d26492970760ca5d33129d2d799e34be5c4782eb
-github.com/aws/aws-sdk-go 90dec2183a5f5458ee79cbaf4b8e9ab910bc81a6
```
Comment (dmcgowan) [inline vendor.conf]: Looks like the new hash is for 1.6.9, that is fine but was this intended?

Comment (mfojtik) [inline vendor.conf]: @dmcgowan sorry, 1.6.9 is what I bumped it to :-) (ETOOMUCHPR's ;-)

**Comment by dmcgowan** [comment]
LGTM

**Comment by nwt** [comment]
LGTM.

**Pull request merged**